amarachukwu-agbo/random-number-generator
# #165145036 Set up eslint

#### What does this PR do?
Set up eslint

#### Description of Task to be completed?
- Install `eslint` and peer dependencies
- Create `eslint` config file

#### How should this be manually tested?
- Clone the repo and install application dependencies.
- Run command  `yarn lint`. 
- You should see the result of the linting on the console.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145036](https://www.pivotaltracker.com/story/show/165145036)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
